src/app/public/modules/text-editor/text-editor.component.scss

```diff
@@ -14,6 +14,9 @@
     border: 1px solid #C0C0C0;
     overflow-y: auto;
     outline: none;
+    &.sky-text-editor-wrapper-disabled {
+      background-color: $sky-theme-modern-background-color-neutral-medium;
```

Comment:
Careful, we're mapping a modern-theme color to a default style declaration here. You might want to find an equivalent non-modern variable for this color here, and then move `$sky-theme-modern-background-color-neutral-medium;` to the modern styles down below.

Reply:
Look like there is no equivalent non-modern variable for `#ededee`. Do I need to create one, or can I just use the exact hex value?

Reply:
Try `$sky-color-gray-07`. If it can't find the variable, you may need to import `@import '~@blackbaud/skyux-design-tokens/scss/variables';`

Reply:
![image](https://user-images.githubusercontent.com/63625452/133451575-b68a8155-73ba-4a6c-88d8-95160e55f36e.png)
I don't see that variable there

Reply:
fixed in commit: ["update background color"](https://github.com/blackbaud/skyux-text-editor/pull/21/commits/c1f823835467c09affe3da756769081b7a92a0f4)

Reply:
fixed (again) in commit: ["correct background color"](https://github.com/blackbaud/skyux-text-editor/pull/21/commits/1036aee6af281ebcc9c9225a5888d384325548e8)